spec/mongoid/query_cache_spec.rb

```diff
@@ -123,6 +123,27 @@
         end
       end
     end
+
+    context "when cache is enabled and second batch of items is requested" do
```

Comment:
You can call this context:
'when query caching is enabled and the batch_size is set'